src/openai/types/audio/transcription_verbose.py

```diff
@@ -0,0 +1,26 @@
+# File generated from our OpenAPI spec by Stainless. See CONTRIBUTING.md for details.
+
+from typing import List, Optional
+
+from ..._models import BaseModel
+from .transcription_word import TranscriptionWord
+from .transcription_segment import TranscriptionSegment
+
+__all__ = ["TranscriptionVerbose"]
+
+
+class TranscriptionVerbose(BaseModel):
+    duration: str
```

Comment:
Shouldn't this be `float` ? See Pydantic warning:
```
...CENSORED.../lib/python3.12/site-packages/pydantic/main.py:390: UserWarning: Pydantic serializer warnings:
  Expected `str` but got `float` with value `118.2699966430664` - serialized value may not be as expected
  return self.__pydantic_serializer__.to_python(
```

Reply:
Thanks, opened an OpenAPI PR https://github.com/openai/openai-openapi/pull/338